examples/simple_worker/main.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import os
+
+from frinx.client.frinx_conductor_wrapper import FrinxConductorWrapper
+from frinx.common.logging import logging_common
+from frinx.common.logging.logging_common import LoggerConfig
+from frinx.common.logging.logging_common import Root
+
+
+def debug_local() -> None:
+    os.environ['UNICONFIG_URL_BASE'] = 'http://localhost/api/uniconfig'
+    os.environ['CONDUCTOR_URL_BASE'] = os.environ.get('CONDUCTOR_URL_BASE', 'http://127.0.0.1:8088/proxy/api')
+    os.environ['SCHELLAR_URL_BASE'] = os.environ.get('SCHELLAR_URL_BASE', 'http://127.0.0.1:3001/query')
+    os.environ['INVENTORY_URL_BASE'] = 'http://localhost/api/inventory'
+    os.environ['INFLUXDB_URL_BASE'] = 'http://localhost:8086'
+    os.environ['RESOURCE_MANAGER_URL_BASE'] = 'http://localhost/api/resource'
+
+
+def register_tasks(conductor_client: FrinxConductorWrapper) -> None:
+    from frinx.workers.schellar.schellar_worker import Schellar
+    from frinx.workers.test.test_worker import TestWorker
+    from frinx.workers.uniconfig.cli_network_topology import CliNetworkTopology
+    from frinx.workers.uniconfig.connection_manager import ConnectionManager
+    from frinx.workers.uniconfig.snapshot_manager import SnapshotManager
+    from frinx.workers.uniconfig.uniconfig_manager import UniconfigManager
+
+    UniconfigManager().register(conductor_client)
+    ConnectionManager().register(conductor_client)
+    CliNetworkTopology().register(conductor_client)
+    SnapshotManager().register(conductor_client)
+    TestWorker().register(conductor_client)
+    Schellar().register(conductor_client)
+
+
+def register_workflows() -> None:
+    logging.info('Register workflows')
+
+    from frinx.workflows.schellar.schellar_workflows import SchellarWorkflows
+    from frinx.workflows.test.test import TestForkWorkflow
+    from frinx.workflows.test.test import TestWorkflow
+
+    TestWorkflow().register(overwrite=True)
+    TestForkWorkflow().register(overwrite=True)
+    SchellarWorkflows().register(overwrite=True)
+
+
+def query_api() -> None:
+    from frinx.common.graphql.client import GraphqlClient
+    from frinx.common.graphql.graphql_types import schema_request
+    from frinx.common.graphql.schema_converter import GraphqlJsonParser
+
+    client = GraphqlClient(endpoint='http://127.0.0.1:3001/query')
```

Comment:
This is the schellar address. Probably not what we want.

`os.environ['SCHELLAR_URL_BASE'] = os.environ.get('SCHELLAR_URL_BASE', 'http://127.0.0.1:3001/query')
`

Reply:
removed from code